lib/logger.ts

```diff
@@ -0,0 +1,62 @@
+import { Middleware } from '../index';
+import { Context } from 'koa';
+
+const COLORS = {
+  blue: '\x1b[34m',
+  green: '\x1b[32m',
+  red: '\x1b[31m',
+  reset: '\x1b[0m'
+};
+
+interface LoggerOptions {
+  colors: {
+    error?: string;
+    success?: string;
+    url?: string;
+    reset?: string;
+  };
+}
+
+interface FinalOptions extends LoggerOptions {
+  colors: {
+    error: string;
+    success: string;
+    url: string;
+    reset: string;
+  };
+}
+
+const defaultOptions: FinalOptions = {
+  colors: {
+    error: COLORS.red,
+    reset: COLORS.reset,
+    success: COLORS.green,
+    url: COLORS.blue
+  }
+};
+
+export default function logger(
+  options?: LoggerOptions
+): Middleware {
+  const finalOptions: FinalOptions = Object.assign(defaultOptions, options);
+  const { colors } = finalOptions;
+
+  return async (
+    ctx: Context,
+    next: () => Promise<any>
+  ): Promise<void> => {
+    const start = Date.now();
+    try {
+      await next();
+    } catch (err) { /* noop */ }
```

Comment:
@scotttrinh was this left over from the original implementation? i assume yes, i forget if this might actually be obscuring actual unhandled errors though

Reply:
Hmm, yeah, I just grabbed this from the original. We probably want to rethrow or something, yeah?

Reply:
Rethrow, but still emit the log.